<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1208</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>791</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>番剧名称</string>
     </attribute>
     <widget class="QTableView" name="tableView_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addButton_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>300</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>添加记录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>340</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>删除选中行</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>531</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>查询</string>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>番剧名称</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="searshButton_3">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>70</y>
         <width>85</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>开始查询</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="showallButton_3">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>300</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>用户信息</string>
     </attribute>
     <widget class="QTableView" name="tableView_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addButton2_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>290</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>添加记录</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>321</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>普通查询</string>
      </property>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_13">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="searshButton_4">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>120</y>
         <width>85</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>开始查询</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="showallButton_4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>330</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>610</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <zorder>tableView</zorder>
   <zorder>tabWidget</zorder>
   <zorder>exitButton</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
